internal/servers/worker/internal/metric/controller_dialer.go

```diff
@@ -0,0 +1,156 @@
+package metric
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/boundary/globals"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/reflect/protoregistry"
+)
+
+const (
+	labelGrpcCode               = "grpc_code"
+	labelGrpcService            = "grpc_service"
+	labelGrpcMethod             = "grpc_method"
+	workerToControllerSubsystem = "worker_to_controller_cluster"
+
+	controllerServersPackage = "controller.servers.services.v1"
```

Comment:
Could you instead use a generated files instead of hardcoding this?  That way, if the generated file gets changed, this would automatically get updated or would result in a compile time error instead of just silently being wrong.

Reply:
You can use the `.Package()` method found [here](https://github.com/hashicorp/boundary/blob/main/internal/gen/controller/servers/services/session_service.pb.go#L930).

This is similar to what I did in the internal/metrics/api.go file [here](https://github.com/hashicorp/boundary/blob/main/internal/servers/controller/internal/metric/api.go#L83).

Reply:
Missed that, thanks!

Reply:
Addressed [e0ff5ce](https://github.com/hashicorp/boundary/pull/1998/commits/e0ff5cee189e8405fb4a99c64adaa5ea7687416f)